src/test/java/com/epam/tc/hw9/testdata/TestData.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.tc.hw9.testdata;
+
+public class TestData {
+    public static User getUserData() {
+        User user = new User();
+        user.setUsername("JohnS");
+        user.setFirstName("John");
+        user.setLastName("Smith");
+        user.setEmail("johns@gmail.com");
+        user.setPassword("admin123");
+        user.setPhone("777777777");
+        return user;
+    }
+
+    public static String requestBodyUser() {
+        User user = new User();
+        String requestBody = String.format(
+            "{\"username\":\"%s\","
+                + "\"firstName\":\"%s\","
+                + "\"lastName\":\"%s\","
+                + "\"email\":\"%s\","
+                + "\"password\":\"%s\","
+                + "\"phone\":\"%s\"}",
+            user.getUsername(), user.getFirstName(), user.getLastName(),
+            user.getEmail(), user.getPassword(), user.getPhone());
+        return requestBody;
+    }
+
+    public static User getUserDataNewPhone() {
```

Comment:
getUserData() and getUserDataNewPhone() are similar methods, so it's better to unify them, to reuse data. In common you can create *.json files with request bodies you need and use them in the appropriate queries. Or try to use JSONObject